src/ui/components/Library/Team/View/Tests/Overview/TestErrorList.tsx

```diff
@@ -0,0 +1,181 @@
+          <LibraryDropdownTrigger
+            onClick={onClickTimeFilter}
+            onKeyDown={onKeyDownTimeFilter}
+            label={filterByTime === null ? "All time" : ""}
+          />
+          {contextMenuTimeFilter}
+        </div>
+        <div>
+          {Object.entries(errorFrequency)
+            .slice(0, MAX_ERRORS_SHOWN)
+            .map(([msg, count], i) => (
+              <ErrorListItem
+                msg={msg}
+                count={count}
+                setSelectedError={setSelectedError}
+                key={i}
+                isSelected={msg === selectedError}
+              />
+            ))}
+        </div>
+      </div>
+      {selectedError ? <ErrorReplays executions={executions} /> : null}
+    </div>
+  );
+}
+
+function ErrorListItem({
+  msg,
+  count,
+  setSelectedError,
+  isSelected,
+}: {
+  msg: string;
+  count: number;
+  setSelectedError: Dispatch<SetStateAction<string | null>>;
+  isSelected: boolean;
+}) {
+  return (
+    <div
+      key={msg}
+      title={msg}
+      className={`flex cursor-pointer flex-row items-center space-x-3 rounded-sm bg-themeBase-100 p-3 ${
+        styles.libraryRow
+      } ${isSelected ? styles.libraryRowSelected : ""}
+`}
+      onClick={() => setSelectedError(msg)}
+    >
+      <div className="flex h-5 w-8 shrink-0 items-center justify-center rounded-md bg-[#F02D5E] text-xs font-bold text-chrome">
+        {count}
+      </div>
+      <div className="overflow-hidden overflow-ellipsis whitespace-nowrap">{msg}</div>
+    </div>
+  );
+}
+
+function ErrorReplays({ executions }: { executions: TestExecution[] }) {
+  const passing = executions.filter(e => e.result === "passed");
+  const failing = executions.filter(e => e.result === "failed");
+  const sortedFailing = orderBy(failing, "createdAt", "desc");
+  const sortedPassing = orderBy(passing, "createdAt", "desc");
+
+  // TODO: show the processed/unprocessed status for the replays -jvv
+  // TODO: add the top commit for the corresponding replay as label -jvv
+  return (
+    <div className="flex flex-col gap-2 py-2 px-4">
+      <div className="flex flex-col gap-2">
+        <div className="overflow-hidden overflow-ellipsis whitespace-nowrap text-lg font-medium">
+          Recent replays that contain this error
+        </div>
+        <div className="flex flex-col gap-2">
+          {sortedFailing.slice(0, MAX_REPLAYS_SHOWN).map((e, i) => (
+            <ErrorReplayList date={e.createdAt} index={i + 1} key={i} status="failed" />
+          ))}
+        </div>
+      </div>
+      <div className="flex flex-col gap-2">
+        <div className="overflow-hidden overflow-ellipsis whitespace-nowrap text-lg font-medium">
+          Recent replays of the test passing
+        </div>
+        <div className="flex flex-col gap-2">
+          {sortedPassing.slice(0, MAX_REPLAYS_SHOWN).map((e, i) => (
+            <ErrorReplayList date={e.createdAt} index={i + 1} key={i} status="passed" />
+          ))}
```

Comment:
We should probably show something for each of these recent blocks when they're empty. This looks kind of broken to me:
<img width="1374" alt="Screen Shot 2023-11-16 at 8 17 26 AM" src="https://github.com/replayio/devtools/assets/29597/9c78cc1b-4596-4614-a694-4f10d46d158d">

Reply:
Recently changed this behavior — for now, I just stopped displaying a block if there are no applicable replays, ie "Recent replays of the test passing" wouldn't show up in the first place.